index.html

```diff
@@ -1,96 +1,171 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="utf-8">
-	<title>Home</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Angelo Deleon</title>
 	<link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+
 	<link rel="stylesheet" href="css/index.css">
+	<script src="./js/Nav.js" async></script>
```

Comment:
Nice job splitting this up to a separate script, but don't use async for something that's going to need to access the DOM right away, it could finish loading before the HTML body, and then it will break when it tries to put event listeners on the DOM